WalletWasabi/Crypto/StrobeProtocol/StrobeHasher.cs

```diff
@@ -26,6 +26,9 @@ public static StrobeHasher Create(string domain)
 		public StrobeHasher Append(string fieldName, IBitcoinSerializable serializable)
 			=> Append(fieldName, serializable.ToBytes());
 
+		public StrobeHasher Append(string fieldName, DateTimeOffset dateTime)
+			=> Append(fieldName, dateTime.ToUnixTimeMilliseconds());
```

Comment:
something about ms unix time feels a bit off to me, but i can't exactly give a strong argument for why it's a problem (because any platform that we expect to port this to should reasonably be able to handle them). i guess the excess precision is the main thing

Reply:
Is the precission that big given:

https://docs.microsoft.com/en-us/dotnet/api/system.datetimeoffset?view=net-5.0#remarks says

> The time component of a DateTimeOffset value is measured in 100-nanosecond units called ticks [..]

If it were nanoseconds I would be suprised as that feels excessive to me but milliseconds are very common these days.

Reply:
yeah fair point